runtime-extension-model/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -1122,6 +1124,13 @@ private void declareConfiguration(ExtensionDeclarer extensionDeclarer) {
         .describedAs("The default correlation id generation expression for every source. This must be DataWeave expression.");
   }
 
+  // See the "abstract-configuration-element" in the mule-core-common.xsd file. It represents the same placeholder.
+  private void declareConfigurationElementsPlaceholder(ConstructDeclarer configuration) {
+    configuration.withOptionalComponent("abstractConfigurationElement")
```

Comment:
add the value of the xsd (https://github.com/mulesoft/mule/blob/master/runtime-extension-model/src/main/resources/META-INF/mule-core-common.xsd#L650) as the description, so the source of truth is the extModel